src/main/java/org/threadly/concurrent/NoThreadScheduler.java

```diff
@@ -53,10 +69,15 @@ public NoThreadScheduler() {
    * Abstract call to get the value the scheduler should use to represent the current time.  This 
    * can be overridden if someone wanted to artificially change the time.
    * 
+   * @param accurate If {@code true} then time estimates are not acceptable
    * @return current time in milliseconds
    */
-  protected long nowInMillis() {
-    return Clock.accurateForwardProgressingMillis();
+  protected long nowInMillis(boolean accurate) {
```

Comment:
maybe just a personal thing but this seems a little pointless for internal uses, could just use the correct Clock calls where you need them since they are statics. would actually be more efficient as you would not need to do a boolean check each time.